monitoring-plugins.spec.in

```diff
@@ -1,86 +1,21 @@
-# Macros
-%define isaix %(test "`uname -s`" = "AIX" && echo "1" || echo "0")
-%define islinux %(test "`uname -s`" = "Linux" && echo "1" || echo "0")
-%define isredhatfamily %(test -f /etc/redhat-release && echo "1" || echo "0")
-
-%if %{isaix}
-	%define _prefix /opt/nagios
-#	%define _defaultdocdir %{_datadir}/doc
-%else
-	%define _libexecdir %{_exec_prefix}/lib/nagios/plugins
-%endif
-%define _sysconfdir /etc/nagios
-
-%define npusr nagios
-%define nphome /opt/nagios
-%define npgrp nagios
-
 Name: monitoring-plugins
 Version: @PACKAGE_VERSION@
 Release: @RELEASE@
-Summary: Host/service/network monitoring program plugins for Nagios and compatible
-
+Summary: Collection of monitoring plugins for Sensu/Icinga/Nagios and other Nagios compatible programs 
 Group: Applications/System
 License: GPL
+
 URL: https://www.monitoring-plugins.org/
-Source0: https://www.monitoring-plugins.org/download/%{name}-%{version}.tar.gz
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Source: https://www.monitoring-plugins.org/download/%{name}-%{version}.tar.gz
 
-%define npdir %{_builddir}/%{name}-%{version}
+Packager: Rob Thomas <xrobau@gmail.com>
```

Comment:
you should not fill `Packager:` field, you won't be the one producing final .rpm